week5_policy_based/atari_util.py

```diff
@@ -1,59 +1,62 @@
 """Auxilary files for those who wanted to solve breakout with CEM or policy gradient"""
 import numpy as np
-import gym
-from scipy.misc import imresize
 from gym.core import Wrapper
 from gym.spaces.box import Box
+from skimage.transform import resize
+
 
 class PreprocessAtari(Wrapper):
-    def __init__(self, env, height=42, width=42, color=False, crop=lambda img: img, 
-                 n_frames=4, dim_order='theano', reward_scale=1,):
+    def __init__(self, env, height=42, width=42, color=False,
+                 crop=lambda img: img, n_frames=4, dim_order='theano'):
```

Comment:
Same here.

Reply:
Nope, here `reward_scale` is used: https://github.com/yandexdataschool/Practical_RL/blob/coursera/week5_policy_based/practice_a3c.ipynb?short_path=bcecb82#L63 (see also #216).